src/controller/cve-id.controller/cve-id.controller.js

```diff
@@ -212,29 +219,38 @@ async function getCveId (req, res, next) {
       return res.status(404).json(error.cveIdNotFound(id))
     }
 
-    const orgUUID = await orgRepo.getOrgUUID(orgShortName) // orgShortName is not null
-    const isSecretariat = await orgRepo.isSecretariatUUID(orgUUID)
+    var finalResult = {}
+    var loggerUuid = 'unauthenticated-user'
+    var orgShortName = ''
+    var orgUUID = null
```

Comment:
Small nitpick, but can you change all the 'var's to 'let's? We should talk about adding eslint to auto handle this in the future.